corehq/apps/hqcase/management/commands/ptop_reindexer_v2.py

```diff
@@ -131,7 +131,14 @@ def handle(self, **options):
         cleanup = options.pop('cleanup')
         noinput = options.pop('noinput')
 
-        for option in ['settings', 'pythonpath', 'verbosity', 'traceback', 'no_color']:
+        for option in [
+            'settings',
+            'pythonpath',
+            'verbosity',
+            'traceback',
+            'no_color',
+            'force_color',
```

Comment:
Where from this param gets passed, is it not an issue in other commands?

Reply:
It would only be a problem on commands that pass all `options` into some other function that fails on unknown arguments. So yes, it is possible that other commands need to be fixed as well.